### DISCUSSION TITLE
Is there a localization guide?

### DISCUSSION THREAD

**Discussion by stone-bits** [apache/superset#13200] [Q&A / Help]
Is there a guide on how to enable localization? 
Or how can we contribute to this?

**Comment by ktmud** [comment]
Hi, @igor-tk , thanks for the interest in contributing.

You can start from here: https://github.com/apache/superset/blob/master/CONTRIBUTING.md#translating

And check out some example PRs: https://github.com/apache/superset/search?q=translation&type=issues
